tests/server_test_cases/person_note_tests.py

```diff
@@ -497,34 +497,56 @@ def assert_params(url=None):
             full_name='_test_given_name _test_family_name',
             details={'Author\'s name:': '_test_author_name'})
 
-        # Now the search should yield a result.
+    @parameterized.expand([(True,), (False,)])
+    def test_search_with_result(self, use_full_text_search):
+        config.set(enable_full_text_search = use_full_text_search)
+        def assert_params(url=None):
+            self.assert_params_conform(
+                url or self.s.url, {'role': 'seek'}, {'ui': 'small'})
```

Comment:
maybe move this def closer to where it's being used (on line 511)?

Reply:
Done.